scripts/ccpp_prebuild.py

```diff
@@ -119,6 +122,33 @@ def setup_logging(debug):
         logging.info('Logging level set to INFO')
     return success
 
+def clean_files(config, static):
+    """Clean files created by ccpp_prebuild.py"""
+    success = True
+    logging.info('Performing clean ....')
+    # Create list of files to remove, use wildcards where necessary
+    files_to_remove = [
+        config['schemes_makefile'],
+        config['schemes_cmakefile'],
+        config['caps_makefile'],
+        config['caps_cmakefile'],
+        config['html_vartable_file'],
+        config['latex_vartable_file'],
+        ]
+    if static:
+        files_to_remove.append(os.path.join(config['caps_dir'], 'ccpp_*_cap.F90'))
+        files_to_remove.append(os.path.join(config['static_api_dir'], '{api}.F90'.format(api=CCPP_STATIC_API_MODULE)))
+    else:
+        files_to_remove.append(os.path.join(config['caps_dir'], '*_cap.F90'))
+        for target_file in config['target_files']:
+            target_file_path = os.path.split(target_file)[0]
+            files_to_remove.append(os.path.join(target_file_path, config['module_include_file'].format(set='*')))
+            files_to_remove.append(os.path.join(target_file_path, config['fields_include_file'].format(set='*')))
+    # Not very pythonic, but the easiest way w/o importing another Python module
+    cmd = 'rm -vf {0}'.format(' '.join(files_to_remove))
+    execute(cmd)
+    return success
+
```

Comment:
Why bother returning a code which cannot be False?

Reply:
Good point. Applies to many more routines in this script.